### ISSUE TITLE
Results of the implementation

### ISSUE THREAD

**Issue by fsk119** [nuclearboy95/Deep-SVDD-Tensorflow#1]
Hi, 
Thanks for your implementaion of the deep svdd. Could you show us the average auc score of your implementation on cifar10. I think it's also very important for others to reuse your work. 

Regards,
fsk

**Comment by nuclearboy95** [comment]
I added main_cifar.py script for an experiment using CIFAR-10 dataset.
Due to some minor differences from the [original implementation](https://github.com/lukasruff/Deep-SVDD-PyTorch), the AUROC using my code (you will immediately get the result by running the file) is not the same as the ones reported by the author.

**Comment by SafiaBahria** [comment]
Can we convert the Deep SVDD implementation in keras so that the whole process inludinh model amd loss calculation is in one library??? Kindly reply Its ent. trying to convert the DeepSVDD in to keras.
